go/teams/teams.go

```diff
@@ -379,18 +379,8 @@ func (t *Team) implicitTeamDisplayName(ctx context.Context, skipConflicts bool)
 	}
 
 	// Add the invites
```

Comment:
This is the function that was previously using AnnotateInvites( to get usernames for keybase-type invites. Now, it just calls UPAKLoader.LookupUsername directly. Behavior should be the same.